src/query_plan/operation.rs

```diff
@@ -857,6 +849,12 @@ pub(crate) mod normalized_fragment_spread_selection {
         pub(crate) selection_set: NormalizedSelectionSet,
     }
 
+    impl NormalizedFragmentSpreadSelection {
+        pub(crate) fn has_defer(&self) -> bool {
+            self.spread.data.directives.has("defer")
```

Comment:
we do store the selection set on the spread selection now so we should also whether selection set has defer

Reply:
good catch, ty!